src/sphinx/main.rst

```diff
@@ -48,6 +48,28 @@ Simply add to the site mappings. In your ``build.sbt`` file:
 
     site.siteMappings <++= Seq(file1 -> "location.html", file2 -> "image.png")
 
+
+Preprocessing with Variable Substitution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The site plugin supports basic variable substitution when copying files from ``src/preprocess``.
```

Comment:
@jsuereth Will fix. Also, I left `PreprocessSite` in the package it started out in. Seem to me like it it might be better under the `site` substructure, but wanted to check with you first. Thoughts?